data-tool/flows/tombstone/tombstone_utils.py

```diff
@@ -826,23 +834,23 @@ def load_data(conn: Connection,
               conflict_error = False,
               expecting_id: bool = True) -> Optional[int]:
     columns = ', '.join(data.keys())
-    values = ', '.join([format_value(v) for v in data.values()])
+    placeholders = ', '.join([f':{key}' for key in data.keys()])
```

Comment:
Fix for `(sqlalchemy.exc.InvalidRequestError) A value is required for bind parameter ...`

Use bind parameters to prevent issues related to special characters like colon `:`. The code still works for value containing single quote `'`.